ad9166.c

```diff
@@ -20,6 +20,18 @@
 #define MAX_CALIB_LENGTH	32
 #define ARRAY_DELIM " [,]"
 
+double default_calibration_freqs[32] = {100000000, 430000000, 510000000, 850000000, 1240000000, 2250000000, 2790000000, 3040000000, 3140000000, 3220000000, 3360000000, 3460000000, 3560000000, 3630000000, 3720000000, 3810000000, 3850000000, 3950000000, 4000000000, 4070000000, 4120000000, 4180000000, 4260000000, 4360000000, 4440000000, 4730000000, 4860000000, 5010000000, 5110000000, 5670000000, 5730000000, 5860000000};
```

Comment:
please split these array into multiple lines and try to fit in 80 characters per line